kolibri/plugins/learn/assets/src/views/learn-page/index.vue

```diff
@@ -44,69 +58,80 @@
 <script>
 
   import { PageNames } from '../../constants';
-  import { getCurrentChannelObject } from 'kolibri.coreVue.vuex.getters';
-  import pageHeader from '../page-header';
-  import contentCardCarousel from '../content-card-carousel';
-  import contentCardGrid from '../content-card-grid';
+  import contentCardGroupCarousel from '../content-card-group-carousel';
+  import contentCardGroupGrid from '../content-card-group-grid';
+  import contentCardGroupHeader from '../content-card-group-header';
   import responsiveWindow from 'kolibri.coreVue.mixins.responsiveWindow';
 
-  const mobileCardNumber = 3;
+  const mobileCarouselLimit = 3;
+  const desktopCarouselLimit = 15;
 
   export default {
     name: 'recommendedPage',
     $trs: {
-      pageHeader: 'Recommended',
       popularSectionHeader: 'Most popular',
       suggestedNextStepsSectionHeader: 'Next steps',
       resumeSectionHeader: 'Resume',
-      popularSectionSubHeader: '{numOfItems, number} popular items',
-      suggestedNextStepsSectionSubHeader: '{numOfItems, number} suggested items',
-      resumeSectionSubHeader: '{numOfItems, number} items to be resumed',
-      featuredSection: 'Featured in {nameOfChannel}',
+      overviewSectionHeader: 'Overview',
     },
     mixins: [responsiveWindow],
     components: {
-      pageHeader,
-      contentCardCarousel,
-      contentCardGrid,
+      contentCardGroupCarousel,
+      contentCardGroupGrid,
+      contentCardGroupHeader,
     },
     computed: {
       isMobile() {
-        return this.windowSize.breakpoint <= 2;
+        return this.windowSize.breakpoint <= 1;
```

Comment:
if you want to have a single definition, you can add `CARD_VIEW_BREAKPOINT = 1` to constants.js and re-use that imported value in both places